src/amo/components/CollectionManager/index.js

```diff
@@ -55,6 +55,7 @@ type State = {|
   description?: string | null,
   name?: string | null,
   slug?: string | null,
+  customSlug?: boolean,
```

Comment:
We should probably also add a default value into the initial state (in the constructor).

Reply:
It will not make difference as we have virtue of Boolean, in every comparison undefined will be `falsey`.

Reply:
That's not very clean though 😉